app/script/view_model/content/CollectionViewModel.js

```diff
@@ -74,6 +75,10 @@ z.viewModel.content.CollectionViewModel = class CollectionViewModel {
     }
   }
 
+  messageRemoved(message) {
+    this.itemRemoved(message.id);
+  }
```

Comment:
See above. I think we need to bind this. Should be at the top of the constructor.